monai/engines/trainer.py

```diff
@@ -185,7 +189,11 @@ def _compute_pred_loss():
             engine.fire_event(IterationEvents.LOSS_COMPLETED)
 
         self.network.train()
-        self.optimizer.zero_grad()
+        if get_torch_version_tuple() < (1, 7, 0):
```

Comment:
`get_torch_version_tuple` always a tuple of length 2. but (1,7) < (1,7,0) is probably not what you want here... please change this to `if get_torch_version_tuple() < (1, 7)`.  could you also help change this https://github.com/Project-MONAI/MONAI/blob/025dee7c89a37675f4f1c3d30173eb5f0d288c3b/monai/handlers/utils.py#L115, although that line seems to have no bug

Reply:
Sure, thanks for your tips, this is an interesting corner case.
I updated in the latest commit.

Thanks.